subprojects/docs/src/docs/userguide/core-plugins/jacoco_plugin.adoc

```diff
@@ -31,7 +31,7 @@ include::sample[dir="snippets/testing/jacoco-quickstart/groovy",files="build.gra
 ====
 
 If the Java plugin is also applied to your project, a new task named `jacocoTestReport` is created.
-By default, a HTML report is generated at `__$buildDir__/reports/jacoco/test`.
+By default, a HTML report is generated at `layout.buildDirectory("reports/jacoco/test")`.
```

Comment:
❓ Shouldn't it be `layout.buildDirectory.dir(`?

Reply:
Yes 🤦